<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>win2</class>
 <widget class="QWidget" name="win2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>736</width>
    <height>431</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Times New Roman</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextBrowser" name="display_d">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>651</width>
     <height>192</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="doc_name">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>300</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>300</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Doctor Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="get_appoint">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>390</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Get Appointment</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="date_time">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>340</y>
     <width>251</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>330</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 12pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Appointment Date &amp; Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>381</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 italic 18pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Available Doctors For Appointments</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 700 14pt &quot;Times New Roman&quot;;</string>
   </property>
   <property name="text">
    <string>Doctors  | |  Qualification</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
